runelite-api/src/main/java/net/runelite/api/AnimationID.java

```diff
@@ -74,6 +74,12 @@
 	public static final int FLETCHING_STRING_MAGIC_LONGBOW = 6689;
 	public static final int FLETCHING_ATTACH_FEATHERS_TO_ARROWSHAFT = 8481;
 	public static final int FLETCHING_ATTACH_HEADS = 8480;
+	public static final int FLETCHING_ATTACH_BOLT_TIPS_TO_BRONZE_BOLT = 8472;
+	public static final int FLETCHING_ATTACH_BOLT_TIPS_TO_IRON_BOLT = 8473;
+	public static final int FLETCHING_ATTACH_BOLT_TIPS_TO_STEEL_BOLT = 8475;
+	public static final int FLETCHING_ATTACH_BOLT_TIPS_TO_MITHRIL_BOLT = 8476;
+	public static final int FLETCHING_ATTACH_BOLT_TIPS_TO_ADAMANT_BOLT = 8477;
+	public static final int FLETCHING_ATTACH_BOLT_TIPS_TO_RUNE_BOLT = 8478;
```

Comment:
Let's put these starting on line 75, after `FLETCHING_STRING_MAGIC_LONGBOW` and before `FLETCHING_ATTACH_FEATHERS_TO_ARROWSHAFT` to be closer to ID order.